src/globus_sdk/experimental/scope_parser/__main__.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import timeit
+import warnings
+
+from ._parser import parse_scope_graph
```

Comment:
I'm not able to run this file locally. This line appears to be the offender.

Reply:
Can you share exactly what's going wrong? I switched to a `__main__.py` from having an `if __name__ == "__main__":` block in the parser module to facilitate easier execution of these tests.
If it's not working for you, then I want to resolve that. The testsuite is all well and good, but I wanted a very clear entry point into this code for us to troubleshoot issues.

In a venv with this branch installed, I can use it like so, including the warning lines in the output:
```
 $ python -m globus_sdk.experimental.scope_parser 'foo[bar[*baz]]'
/home/sirosen/dev/sdk-and-cli/sdk/src/globus_sdk/experimental/scope_parser/__main__.py:7: UserWarning: This is a test module for scope parsing. It is not intended for public use.
  warnings.warn(
top level scopes: foo
digraph scopes {
  rankdir="LR";

  foo

  foo -> bar;
  bar -> baz [ label = "optional" ];

}
```

What happens when you try that same usage?